trackbasket_be/resources/shopping_list.py

```diff
@@ -53,12 +53,13 @@ def get(self,id):
   def patch(self,id):
     request_data = request.json
     at_risk_user = AtRiskUser.query.filter_by(at_risk_user_id=id).first()
-    store = at_risk_user.store[0]
     if at_risk_user is None:
       return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "At risk user #{} not found".format(id)} } }, 400
+    store = at_risk_user.store[0]
+    if at_risk_user.shopping_lists == []:
+        return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "No shopping list associated with this user"} } }, 400
     else:
```

Comment:
returns an error message if trying to update a `shopping list` for a user who doesn't have any